app/views/groups/_add_member_modal.html.erb

```diff
@@ -30,15 +30,27 @@
         <% end %>
       </div>
       <div class="mx-auto text-center p-3">
+        <p>OR USE</p>
+        <%= button_to generate_token_group_path,{remote: true,method: :put,class: 'btn primary-button', 'data-toggle':'collapse', 'href':'#collapseExample', 'aria-expanded':'false', 'aria-controls':'collapseExample', id: "clipboard_b"} do %>
         <p><%= t("groups.use_alternative") %></p>
         <%= button_to generate_token_group_path,{remote: true,method: :put,class: 'btn primary-button', 'data-toggle':'collapse', 'href':'#collapseExample', 'aria-expanded':'false', 'aria-controls':'collapseExample'} do %>
           <i class="fas fa-external-link-alt"></i>
           <span><%= t("groups.invite_link_btn") %></span>
         <% end %>
         <div class="collapse" id="collapseExample">
           <textarea id="embedInviteLink" class="group-invite-link-text" rows="4" cols="35"></textarea>
+          <small class="projectshow-embed-text-confirmation">Copied to Clipboard!</small>
```

Comment:
Refactor css for better naming and make the text translatable using I18n

Reply:
@tachyons "projectshow-embed-text-confirmation" classname is already thier in embed button, I just use it here.

Reply:
That is the issue, that not be refactored, as the same class need to be used outside project show page as well

Reply:
So I need to create new class for it and style it wisely?

Reply:
`projectshow-embed-text-confirmation' This class has nothing to do with project show page. So rename it and also update the original usage

Reply:
Okhay so I have to rename the class to 'embed-text-confirmation' and update both the files where the class is being used.